Dockerfile-wolfi

```diff
@@ -0,0 +1,134 @@
+ARG BASE_IMAGE=docker.io/library/ubuntu:24.04@sha256:278628f08d4979fb9af9ead44277dbc9c92c2465922310916ad0c46ec9999295
+####################################################################################################
+# Builder image
+# Initial stage which pulls prepares build dependencies and CLI tooling we need for our final image
+# Also used as the image in CI jobs so needs all dependencies
+####################################################################################################
+FROM docker.io/library/golang:1.23.3@sha256:73f06be4578c9987ce560087e2e2ea6485fb605e3910542cadd8fa09fc5f3e31 AS builder
```

Comment:
The version matches https://github.com/argoproj/argo-cd/blob/b3916428c130768dbd010e2fc4691d500a98e25e/Dockerfile#L7, but the sha is different. Is this expected? Can we make them match?

Reply:
I can make them match if you like. I just pulled the newest SHA, so this one is probably slightly ahead of the main Dockerfile.

Really there should be some automation to update them e.g. https://github.com/chainguard-dev/digestabot